admin_guide/manage_rbac.adoc

```diff
@@ -762,8 +762,158 @@ To bind the new role to a user, run:
+----
+
+The following excerpt from the help of this command describes its usage:
+
+----
+Create a ClusterRole.
+
+Usage:
+  oc create clusterrole NAME --verb=verb --resource=resource.group [--resource-name=resourcename] [--dry-run] [flags]
+
+Examples:
+  # Create a ClusterRole named "pod-reader" that allows user to perform "get", "watch" and "list" on pods
+  oc create clusterrole pod-reader --verb=get,list,watch --resource=pods
+  
+  # Create a ClusterRole named "pod-reader" with ResourceName specified
+  oc create clusterrole pod-reader --verb=get --resource=pods --resource-name=readablepod --resource-name=anotherpod
+  
+  # Create a ClusterRole named "foo" with API Group specified
+  oc create clusterrole foo --verb=get,list,watch --resource=rs.extensions
+  
+  # Create a ClusterRole named "foo" with SubResource specified
+  oc create clusterrole foo --verb=get,list,watch --resource=pods,pods/status
+  
+  # Create a ClusterRole name "foo" with NonResourceURL specified
+  oc create clusterrole "foo" --verb=get --non-resource-url=/logs/*
+  
+  # Create a ClusterRole name "monitoring" with AggregationRule specified
+  oc create clusterrole monitoring --aggregation-rule="rbac.example.com/aggregate-to-monitoring=true"
+
+Options:
+      --aggregation-rule=: An aggregation label selector for combining ClusterRoles.
+      --allow-missing-template-keys=true: If true, ignore any errors in templates when a field or map key is missing in
+the template. Only applies to golang and jsonpath output formats.
+      --dry-run=false: If true, only print the object that would be sent, without sending it.
+      --non-resource-url=[]: A partial url that user should have access to.
+  -o, --output='': Output format. One of:
+json|yaml|wide|name|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...
+See custom columns [http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns], golang template
+[http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template
+[http://kubernetes.io/docs/user-guide/jsonpath].
+      --resource=[]: Resource that the rule applies to
+      --resource-name=[]: Resource in the white list that the rule applies to, repeat this flag for multiple items
+      --save-config=false: If true, the configuration of current object will be saved in its annotation. Otherwise, the
+annotation will be unchanged. This flag is useful when you want to perform kubectl apply on this object in the future.
+      --template='': Template string or path to template file to use when -o=go-template, -o=go-template-file. The
+template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].
+      --validate=false: If true, use a schema to validate the input before sending it
+      --verb=[]: Verb that applies to the resources contained in the rule
+
+----
+
+[[creating-derivative-cluster-role]]
+== Creating a cluster role from another cluster role
```

Comment:
I can't think of a reason why someone would do this. Is this information you have? Can that be included?

Reply:
Remove the YAML please.  The create command is the correct approve.